.circleci/config.yml

```diff
@@ -37,6 +41,20 @@ jobs:
       TRAVIS_OS_NAME: osx
     steps: *ci_steps
 
+  sync_docs_s3:
+    machine: true
+    steps:
+      - attach_workspace:
+          at: /tmp/workspace
+      - run: |
+          echo $CIRCLE_SHA1 > /tmp/workspace/docs/revision.txt
+      - run: |
+          docker run -v /tmp/workspace/docs:/docs \
+            -e AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID} \
+            -e AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY} \
+            mesosphere/aws-cli \
+            s3 sync /docs/ s3://crystal-api/api/master
```

Comment:
Add `--delete` flag to remove files that are no longer in the genetated docs.

Reply:
Good catch. Adding it (and checkin the output in the next commit with a dryrun, will remove shortly)